tiledb/sm/query/sparse_global_order_reader.cc

```diff
@@ -942,6 +942,7 @@ Status SparseGlobalOrderReader::copy_var_data_tiles(
                 0,
                 rcs.length_,
                 cell_offsets[i]);
+        (void)dest_cell_offset;
```

Comment:
FYI, had to add these to compile on GCC 7.3, otherwise got unused variable warning -> Werror.